HW1/P2/P2.txt

```diff
@@ -1 +1,7 @@
 # Your discussion here
+The result of P2a is reasonable. It failed to produce a very handy partition. Because for the 2000*2000 pixels, some of the pixel's greyscale value was calculated and drew many times, while some of them might never been touched. So it is a bad idea to partition them in the default way.
```

Comment:
I don't think that the pixel values were "calculated and drew" many times...some pixels just required many more iterations than others and were drawn once. Every pixel was touched, or else we would not have created the mandelbrot set! I agree that it is a bad idea to partition in the default way.